CONTRIBUTING.md

```diff
@@ -1,7 +1,7 @@
-If you find a bug or you'd like to propose a feature, please feel free to raise an issue on our [issue tracker](https://github.com/input-output-hk/cardano-wallet/issues).
+If you find a bug or you'd like to propose a feature, please feel free to raise an issue on our [issue tracker](https://github.com/cardano-foundation/cardano-wallet/issues).
 
 Pull requests are welcome!
 
 When creating a pull request, please make sure that your code adheres to our [coding standards](https://github.com/input-output-hk/adrestia/blob/master/docs/code/Coding-Standards.md).
 
-For more information, please consult our [contributors' guide](https://input-output-hk.github.io/cardano-wallet/contributing).
+For more information, please consult our [contributors' guide](https://cardano-foundation.github.io/cardano-wallet/contributing).
```

Comment:
```suggestion
For more information, please consult our [Contributor Manual](https://cardano-foundation.github.io/cardano-wallet/contributor).
```

The [Contributor Manual][1] has been rearranged.

  [1]: https://cardano-foundation.github.io/cardano-wallet/contributor